frontend/simple/views/Invite.vue

```diff
@@ -103,13 +103,11 @@
 import * as Events from '../../../shared/events'
 import backend from '../js/backend/'
 import { latestContractState } from '../js/state'
-import { HapiNamespace } from '../js/backend/hapi'
+import { namespace } from '../js/backend/hapi'
 import L from '../js/translations'
 
-const namespace = new HapiNamespace()
-
 export default {
-  name: 'InviteView',
+  name: 'Invite',
```

Comment:
This is known issue to be encompassed in the upcoming invite/voting changes. We didn't have a way of knowing who was in a group when this was created(other than the founder).